src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -259,7 +262,7 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         private delegate void WebViewHitTestFinishedProxyCallback(IntPtr data);
 
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        private delegate void WebViewResponsePolicyDecidedCallbackDelegate(IntPtr maker);
+        private delegate void WebViewPolicyDecidedCallbackDelegate(IntPtr maker);
```

Comment:
could you remove `Delegate` in API's name?
(refer) https://code.sec.samsung.net/confluence/display/SE/2.9+Delegate%2C+Event

Reply:
Done.